lib/Home.dart

```diff
@@ -120,11 +120,21 @@ class _HomeState extends State<Home> {
 class SignOutButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return RaisedButton(
-      child: Text('Sign out'),
-      onPressed: () {
-        googleSignIn.signOut();
-      },
+    return SizedBox(
+      width: double.maxFinite,
+      height: 100,
+      child: RaisedButton(
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Text('Sign out',
+                textAlign: TextAlign.start),
+          ],
+        ),
+        onPressed: () {
+          googleSignIn.signOut();
+        },
+      ),
```

Comment:
Why not just use a RaisedButton with a Text child?

Reply:
I used a SizedBox so it accessible on the emulator. Otherwise, it would not have been above to click it.

Reply:
Okay, but instead of doing this inside the actual widget definition, could you just do it one level out? Like wherever the SignOut Button is used, put that code around it instead of putting it inside the SignOut Button's build.